### ISSUE TITLE
Is there a C# API to use ImageReader of CNTK ?

### ISSUE THREAD

**Issue by veboddu** [Microsoft/CNTK#2955]
Is there C# API to use transform functions of  "CNTK/Source/Readers/ImageReader/ImageTransformers.cpp" available? 

Currently in [Understanding and Extending Readers](https://docs.microsoft.com/en-us/cognitive-toolkit/understanding-readers) explains only for python.

Can you please point to an example on how to use these Readers with C# CNTK API

**Comment by KeDengMS** [comment]
Yes, please use ImageDeserializer. Samples can be found [here](https://github.com/Microsoft/CNTK/blob/06ee7b83a5a95457f2e6e1fd8a452f28dc47ebd0/Examples/TrainingCSharp/Common/CifarResNetClassifier.cs#L258-L268).
